recipes-mender/artifacts/artifacts.bb

```diff
@@ -0,0 +1,50 @@
+DESCRIPTION = "Mender image artifact library"
+GO_IMPORT = "github.com/mendersoftware/artifacts"
+
+inherit go
+SRC_URI = "git://github.com/mendersoftware/artifacts.git;protocol=https"
+
+SRCREV = "${AUTOREV}"
+PVBASE := "${PV}"
+PV = "${PVBASE}+git${SRCPV}"
+
+LICENSE = "Apache-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=e3fc50a88d0a364313df4b21ef20c29e \
+                    file://LIC_FILES_CHKSUM.sha256;md5=ca079eb955443fa5828872744095fada \
```

Comment:
Technically the `LIC_FILES_CHKSUM.sha256` covers the `LICENSE` file as well, so both are not necessary.